grants_tagger/models/bert_mesh.py

```diff
@@ -0,0 +1,77 @@
+from transformers import BertTokenizerFast
+from torch.utils.data import Dataset, DataLoader
+from tqdm import tqdm
+import scipy.sparse as sp
+import numpy as np
+import torch
+
+from grants_tagger.bertmesh.model import BertMesh
+
+
+class MeshDataset(Dataset):
+    def __init__(self, X):
+        self.tokenizer = BertTokenizerFast.from_pretrained("bert-base-uncased")
```

Comment:
Should the tokeniser size be a parameter?

I think it's called `max_position_embeddings`

Reply:
what do you mean? I am not familiar with it